klasses/bootcamp_admissions_client_test.py

```diff
@@ -140,11 +142,26 @@ def test_json_raises(test_data, mocked_get_200):
 
 def test_can_pay_klass(test_data, mocked_get_200):
     """
-    Test BootcampAdmissionClient.can_pay_klass
+    Test BootcampAdmissionClient.can_pay_klass with a mix of legacy and fluidreview klasses
     """
     user, _ = test_data
+    WebhookRequestFactory(award_id=19, user_email=user.email, user_id=user.profile.fluidreview_id)
 
     boot_client = BootcampAdmissionClient(user)
     assert boot_client.can_pay_klass(16) is True
+    assert boot_client.can_pay_klass(19) is True
     assert boot_client.can_pay_klass(12) is False
     assert boot_client.can_pay_klass('foo') is False
+
+
+def test_can_pay_klass_webhook_only(test_data, mocked_requests_get):
+    """
+    Test BootcampAdmissionClient.can_pay_klass when legacy admissions API is unavailable
+    """
+    mocked_requests_get.side_effect = ConnectionError
+    user, _ = test_data
+    WebhookRequestFactory(award_id=19, user_email=user.email, user_id=user.profile.fluidreview_id)
```

Comment:
Can you put `award_id` in a variable and reference it below as well?

Reply:
✅